oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -52,6 +52,27 @@ This proposal focuses on long-term developer *Documentation* needs. It does not
 Decisions
 *********
 
+* We will use `reStructuredText (rST)`_ as our default documentation format for all developer documentation except for the exceptions outlined here. See the :ref:`Format Rationale` section below for more info on why RST.
+
+  * Python docstrings that are going to be renderd in an `Open API`_ context should be formatted in Markdown.
```

Comment:
```suggestion
  * Python docstrings that are going to be rendered in an `Open API`_ context should be formatted in Markdown.
```

Reply:
We could add a bullet here for JSDoc strings using Markdown - there's tooling to parse it, but we were unable to find an RST equivalent.